examples/skymapStitch.py

```diff
@@ -187,11 +186,25 @@ def getSkyMapCoaddUtil(ctrCoord, width, height, filt, source, mapType, patchType
     source = "/lsst7/releaseW13EP"
     mapType = "deepCoadd_skyMap"
     patchType = "deepCoadd"
+
+    #raA = afwGeom.Angle(0.0, afwGeom.degrees)
+    #decA = afwGeom.Angle(0.01667, afwGeom.degrees)
+    #ctrCoord = afwCoord.Coord(raA, decA, 2000.0)
```

Comment:
why are these 3 lines here? Are they meant to be describing an alternative constructor for the example? If so I'd prefer an extra line of commentary indicating this.